protocol/chat.go

```diff
@@ -35,6 +35,11 @@ const (
 	ChatTypeCommunityChat
 )
 
+const (
+	FirstMessageTimestampUndefined = 0
+	FirstMessageTimestampNoMessage = 1
```

Comment:
I wonder if this is really needed. I'm probably missing something here.. Can't we work with only `0` and assume that no messages have been saved (yet)? So it's either `0` or `> 0` ?

Reply:
Is is needed, because of backward compatibility. It is to distinguish if the admin actually propagated the first message timestamp or not.
If there is an existing community chat that has messages to fetch and the admin hasn't logged in yet, then members would not be able to fetch these messages if there is only `0` state.